.env

```
@@ -0,0 +1,7 @@
+# Host ports remapped to avoid clashes with chati (3000) and grafana (3001)
+# Container-internal ports are unchanged.
+REDIS_PORT=6380
+WEB_PORT=3005
+API_PORT=3006
+NEXT_PUBLIC_API_BASE_URL=http://localhost:3006/api/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_BASE_URL key should go before the REDIS_PORT key

(UnorderedKey)